index.js

```diff
@@ -80,22 +80,39 @@ var WebDriverInstance = function (baseBrowserDecorator, args, logger) {
     log.debug('WebDriver config: ' + JSON.stringify(config));
     log.debug('Browser capabilities: ' + JSON.stringify(spec));
 
-    self.driver = wd.remote(config, 'promiseChain');
-    self.browser = self.driver.init(spec);
+    self.browser = wd.remote(config);
 
     var interval = args.pseudoActivityInterval && setInterval(function() {
       log.debug('Imitate activity');
       self.browser.title();
     }, args.pseudoActivityInterval);
 
-    self.browser
-        .get(url)
-        .done();
+    // With the "Promise-based" API from "wd", you can't actually handle
```

Comment:
This comment is fine in the PR description but delete it here in the code. It explains the change, not the code.

Reply:
> This comment is fine in the PR description but delete it here in the code. It explains the change, not the code.

I disagree.  It explains why the code must be in this form.  It would be easy for someone who isn't reading the commit history to change this back to the more compact syntax.  The comment ensures that readers of the code do not do this.